### ISSUE TITLE
store._normalizeTypeKey not being used in canary - file concat issue?

### ISSUE THREAD

**Issue by jmurphyau** [emberjs/data#2990]
Works in beta.

[Beta JSBin](http://emberjs.jsbin.com/meteyanune/2/edit?js,output)
[Canary JSBin](http://emberjs.jsbin.com/qelewenicu/21/edit?js,output)

I was try to look for [`store._normalizeTypeKey`](https://github.com/emberjs/data/blob/master/packages/ember-data/lib/system/store.js#L1950) in my source code (Chrome Inspector) but couldn't seem to find it anywhere - all I could find was references to a function [`ember$data$lib$system$normalize$type$key$$default`](https://github.com/components/ember-data/blob/canary/ember-data.js#L4623) which isn't attached to a store - it's a stand alone function.. This function looks like its only in the [components/ember-data](https://github.com/components/ember-data/tree/canary) repo..

Looks like something strange with import statements and file concatenation thats doing this..?

**Comment by wecc** [comment]
I think this is something that is done automatically in the build process, probably for optimizations when minifying (speed, size? not really sure).

As you can see in the `_normalizeTypeKey` source it only contains `return camelize(singularize(key)); ` and has no references to the store so I assume that the build system is smart enough to figure that out and know how to optimize it.

Are you experiencing any issues related to this or is it just that you're unable to find the method name in the source?

**Comment by jmurphyau** [comment]
It came about because of [this stack overflow question](http://stackoverflow.com/questions/29543803/are-single-lettered-word-models-supported-in-emberjs/29574457) I wanted to answer 1 which I also have an issue relating to but have been putting it off - one of those small issues that's not really _too_ important.. 

My issue isn't around single letter words in a model name it's more around a model name having multiple capital letters.. 'genDNGroup' - which is Gen DN Group.. DN standing for directory number..

**Comment by igorT** [comment]
@jmurphyau https://github.com/components/ember-data/blob/canary/ember-data.js#L11257 is this the method you were looking for? Btw enabling source maps in Chrome will make these function names nicely formatted. Not really sure what to do about multiple capital letters, can you open an issue for that? Thanks!

**Comment by jmurphyau** [comment]
@igorT my links (and yours) all look to reference 'canary' rather than a particular commit - all moving targets.. I just clicked my links and they don't point to where I intended anymore.. But basically that function didn't exist at one point but yeah it is back again now..

Additionally - is it worth opening an issue for multiple capital letters? I believe there is some work underway to move everything to dasherized format everywhere? this would solve the issue as genDNGroup wouldn't appear anywhere and gen-dn-group would be the only thing used everywhere..

If that work isn't going on/planned, I'll open an issue - yep.
